package_test.go

```diff
@@ -22,25 +22,40 @@ type PackageSuite struct{}
 
 var _ = Suite(&PackageSuite{})
 
-func setupDB() (*sql.DB, error) {
-	return sql.Open("sqlite3", "file:test.db?cache=shared&mode=memory")
-}
+func (s *PackageSuite) TearDownTest(c *C) {
+	// Try to run finalizers by calling GC several times.
+	for i := 0; i <= 10; i++ {
```

Comment:
It should technically only need running twice if the value is out of scope (once to garbage collect it and the second time to run the actual finalizer). But I guess there's no harm in running it a few more times.

Reply:
I was treating the finalizers as an implementation detail, hence why running them more times than needed to avoid breaking the tests in the future.